notebooks/test_output/Indices/YYYY_MM_DD_Celeste_Adaptation_16-21_indices.txt

```diff
@@ -671,99 +671,99 @@ katharo.ADAPT.21.H12.240000.0	iTru5_plate	384LDV_AQ_B2	P24	250	iTru5_124_H	AACGC
 43R.Month6.7	iTru7_plate	384LDV_AQ_B2	G17	250	iTru7_104_09	AGAGCCTT	55197	IndexPCRPlate	P19
 43R.Month6.8	iTru7_plate	384LDV_AQ_B2	G19	250	iTru7_104_10	GCAATGGA	55198	IndexPCRPlate	P21
 43R.Month6.9	iTru7_plate	384LDV_AQ_B2	G21	250	iTru7_104_11	CTGGAGTA	55199	IndexPCRPlate	P23
-46L.Month6.1	iTru7_plate	384LDV_AQ_B2	G23	250	iTru7_104_12	GAACATCG	55200	IndexPCRPlate	B2
```

Comment:
Ummm, I am lost here, and I think I am going to need you to walk me through what you are seeing in more detail.  To me it appears the only differences between the file in the PR and the file in master is in the `Sample` column, where some of the names have changed.  It looks to me like every sample has two different entries, one for the iTru5_plate and one for the iTru7_plate; in the example you highlighted, the name `46L.Month.6.1` has been replaced by `20001CC.B.FW2`, but other than that, the index assignments haven't changed since `20001CC.B.FW2` has the same iTru5 and iTru7 indices as `46L.Month.6.1` used to:

<img width="1320" alt="Screenshot 2024-08-21 at 3 23 49 PM" src="https://github.com/user-attachments/assets/19a76fa9-5083-4578-96a7-357465facc4e">

<p />
<p />

<img width="1313" alt="Screenshot 2024-08-21 at 3 24 07 PM" src="https://github.com/user-attachments/assets/29605c41-14a2-409f-83ea-56f40c94f716">

Can you give me more clarity on what you're seeing?

Reply:
Ah ty! Yes, you are right. Once I pulled both files out and reviewed them it all became clear.